packages/compose/src/hooks/use-focus-outside/test/index.js

```diff
@@ -85,56 +95,4 @@ describe( 'useFocusOutside', () => {
 
 		expect( mockOnFocusOutside ).toHaveBeenCalled();
 	} );
-
-	it( 'should not call handler if focus shifts outside the component when the document does not have focus', async () => {
```

Comment:
This test is an implementation detail on useFocusOutside which is no longer valid. We don't test for document.hasFocus()